spec/test_enumerables.rb

```diff
@@ -0,0 +1,297 @@
+# spec/test_enumerables.rb
+
+# frozen_string_literal: true
+
+require '../main.rb'
```

Comment:
Try changing this to ```require_relative  '../main.rb'``` to fix the error when it's ```require '../main.rb'```

Reply:
Solved!

Thank you very much.